Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -227,13 +226,12 @@ public static bool AreFoldersAvailable
             }
         }
 
-        static MixedRealityToolkitFiles()
-        {
-            Init();
-        }
-
         private static void Init()
         {
+            // Note that this file used to have an InitializeOnLoad handler to handle
+            // early initialization of the folder refresh. However, this had an effect of slowing down
```

Comment:
Are we going to need a menu item for manual initialization like the others?

Reply:
Good question! Nope - in this case since the other functions already handle this lazy-init, the menu based thing isn't necessary.